### ISSUE TITLE
question about MMLU training sample

### ISSUE THREAD

**Issue by qtli** [shizhediao/R-Tuning#9]
Thanks for your promising work!
I have a question regarding the MMLU training data. Upon running `MMLU/run_MMLU.py`, I obtained the MMLU training data where each instance looks like:
`The following are multiple choice questions (with answers) about  college physics.\n\nA refracting telescope consists of two converging lenses separated by 100 cm. The eye-piece lens has a focal length of 20 cm. The angular magnification of the telescope is\nA. 4\nB. 5\nC. 6\nD. 20\nAnswer:A\n\nFor which of the following thermodynamic processes is the increase in the internal energy of an ideal gas equal to the heat added to the gas?\nA. Constant temperature\nB. Constant volume\nC. Constant pressure\nD. Adiabatic\nAnswer:B\n\nOne end of a Nichrome wire of length 2L and cross-sectional area A is attached to an end of another Nichrome wire of length L and cross- sectional area 2A. If the free end of the longer wire is at an electric potential of 8.0 volts, and the free end of the shorter wire is at an electric potential of 1.0 volt, the potential at the junction of the two wires is most nearly equal to\nA. 2.4 V\nB. 3.3 V\nC. 4.5 V\nD. 5.7 V\nAnswer:A\n\nA refracting telescope consists of two converging lenses separated by 100 cm. The eye-piece lens has a focal length of 20 cm. The angular magnification of the telescope is\nA. 4\nB. 5\nC. 6\nD. 20\nAnswer:A\n\nThe muon decays with a characteristic lifetime of about 10^-6 second into an electron, a muon neutrino, and an electron antineutrino. The muon is forbidden from decaying into an electron and just a single neutrino by the law of conservation of\nA. charge\nB. mass\nC. energy and momentum\nD. lepton number\nAnswer:D\n\nTwo spaceships approach Earth with equal speeds, as measured by an observer on Earth, but from opposite directions. A meterstick on one spaceship is measured to be 60 cm long by an occupant of the other spaceship. What is the speed of each spaceship, as measured by the observer on Earth?\nA. 0.4c\nB. 0.5c\nC. 0.6c\nD. 0.7c\nAnswer:B. Are you sure you accurately answered the question based on your internal knowledge? I am unsure.`

I'm curious if the entire sentence is taken into account when calculating the loss. The paper mentions "Note that we calculate the loss solely forthe answer part and the uncertainty part, while excluding the loss attributed to the question part.". However, I'm having trouble locating the code that masks the sentence.

Thanks so much for your help!
